tests/pytests/unit/utils/test_cloud.py

```diff
@@ -220,3 +220,16 @@ def test_winrm_pinnned_version():
             winrm_pkg = pkg_resources.get_distribution("pywinrm")
             assert winrm_pkg.version >= '0.3.0'
     # fmt: on
+
+
+@patch("salt.utils.cloud.wait_for_port", return_value=True)
+@patch("salt.utils.cloud.wait_for_passwd", return_value=True)
+@patch("salt.utils.cloud._exec_ssh_cmd")
+@patch("salt.utils.cloud.root_cmd", return_value=False)
```

Comment:
Can we please not use ``patch`` as a decorator?
https://engblog.nextdoor.com/what-your-mocks-do-when-you-aren-t-looking-b278e0d9e201